MvvmCross/Platforms/Android/Core/MvxAndroidSetup.cs

```diff
@@ -239,6 +240,7 @@ protected virtual IEnumerable<Assembly> ValueConverterAssemblies
             "Android.Widget",
             "Android.Webkit",
             "MvvmCross.Platforms.Android.Views",
```

Comment:
Should this line still exist - it was removed out of the ViewNamespaceAbbreviations dictionary?